### DISCUSSION TITLE
Standard adókódok

### DISCUSSION THREAD

**Discussion by zoltangergely** [nav-gov-hu/eVAT#25] [Ötletek - gépi interfész]
A jelenlegi kódrendszer -legalább is számomra- nem sugallja, hogy melyik fő sorhoz kell hozzáadni a részletező sort. Erre a módszerre ajánlásokat a főkönyvi számlakeret felépítéséről szóló középiskolás közgazdászképzés számviteli könyveiben lehet olvasgatni.
Továbbá: a jelenlegi Excel táblában a 65-ös oldalaira és soraira történik hivatkozás, amelyek azonban az ÁNYK rendszerben évente változhatnak. Hogy ez nem történt meg, annak az az oka, hogy lényegi változás nem történt már harmadik éve a hatvanötösökben. Tessék áttekinteni a kapcsolódó XSD-t, ahol látszik, hogy minden mezőnek van fidje, vidje, meg didje. Na ez utóbbiak az imp import fájlok azonosítói, a fidek meg az xkáerek eazonjai. A videk meg a nyomtatvány kinézete szerinti, oldal- és sorspecifikus attributumok. Jó lenne a standard adókódokat didelni.

**Comment by NTCA-HM** [comment]
Kedves @zoltangergely !

A standard adókód lista az Áfa tv.-ből levezethető összes bevallás releváns ügyleteket tartalmazza. Jellemzően egy-egy ügyfélnek ennél lényegesen rövidebb a saját maga által használt adókód listája. Éppen ezért érdemes a könyvelő program ügyféli testreszabásánál a vállalkozás által használt adókódokat összerendelni a standard adókód párjával és nem feltétlenül hasznos a teljes standard adókódlistát implementálni az ügyféli rendszerbe. Az adókódok megfeleltetésénél érdemes gondolni arra, hogy a NAV oldaláról a standard adókódok időben változhatnak. A standard adókódok változása, ha érint ügyféli vállalati adókódot, akkor annak kezelése mindenképpen kliens oldali kötelezettséget jelent. Érvényét vesztett standard adókód használata esetén a data XML-ből nem képezhető áfabevallás.
